### ISSUE TITLE
ACS FWTS ACPI DBG2 failures

### ISSUE THREAD

**Issue by samerhaj** [pftf/RPi4#69]
From the ACS 1.4 tests on Rel 1.15

https://github.com/pftf/acs-reports/blob/master/v1.15/acs-enterprise-v2.4/luv-results-2020-06-12--16-30-39/parsed/fwts

```
1.1  fwts FAILED INFO [HIGH] DBG2DeviceNotFound: Test 1, DBG2 Device '\_SB.URT0' not found in
1.1  fwts ACPI INFO object name space.
```

**Comment by samerhaj** [comment]
The issue is that DBG2 is referring to '_SB.URT0', but the actual namespace for the device is '_SB_.GDV0.URT0'

Submitting a fix

**Comment by samerhaj** [comment]
Fixed in v1.19

**Comment by samerhaj** [comment]
Confirmed fixed in ACS 2.5 beta:

https://github.com/pftf/acs-reports/blob/master/v1.19/acs-enterprise-v2.5-beta/luv-results-2020-08-24--10-11-06/raw/fwts#L1659

`1.1  fwts PASSED: INFO Test 1, No issues found in DBG2 table.
1.1  fwts Test INFO 2 of 2: DBG2 ARM SBSA Generic UART test,
1.1  fwts PASSED: INFO Test 2, DBG2 provides a standard serial debug port and describes ARM
1.1  fwts SBSA INFO Generic UART`
